firmware/brake/kia_soul_ps/brake_control_module.ino

```diff
@@ -641,170 +678,245 @@ void waitEnter()
+// *****************************************************
+void brake_exit( )
 {
     // close master cylinder solenoids
-    smc.solenoidsOpen();
+    smc.solenoidsOpen( );
 
     // depower wheel solenoids to vent brake pressure at wheels
-    brakes.depowerSLA();
+    brakes.depowerSLA( );
 
     // unswitch brake switch
     digitalWrite( PIN_BRAKE_SWITCH_1, HIGH );
     digitalWrite( PIN_BRAKE_SWITCH_2, HIGH );
 }
 
 
-//
-static void check_rx_timeouts( void )
+// *****************************************************
+// Function:    check_rx_timeouts
+// 
+// Purpose:     If the control is currently enabled, but the receiver indicates
+//              a "watchdog" timeout, then disable the control
+// 
+// Returns:     void
+// 
+// Parameters:  None
+// 
+// *****************************************************
+static void check_rx_timeouts( )
 {
     // local vars
-    uint32_t delta = 0;
-
-    // get time since last receive
-    get_update_time_delta_ms(
-			rx_frame_ps_ctrl_brake_command.timestamp,
-			GET_TIMESTAMP_MS(),
-			&delta );
+    uint32_t delta = 
+    	timer_delta_ms( rx_frame_ps_ctrl_brake_command.timestamp );
 
-    // check rx timeout
     if( delta >= PS_CTRL_RX_WARN_TIMEOUT )
     {
         // disable control from the PolySync interface
-        if( controlEnabled )
+        if( control_enabled )
         {
-            Serial.println("control disabled: timeout");
-            controlEnabled = false;
-            brakeStateMachine.transitionTo(Wait);
+            Serial.println( "control disabled: timeout" );
+            control_enabled = false;
+            brakeStateMachine.transitionTo( Wait );
         }
     }
 }
 
 
-// the setup routine runs once when you press reset:
+/* ====================================== */
+/* ================ SETUP =============== */
+/* ====================================== */
+
+// *****************************************************
+// Function:    setup
+// 
+// Purpose:     Initialize and clear all global data
+//              Set up hardware
+//              Initialize control loop variables
+//				Setup routine runs once you press reset
+// 
+// Returns:     void
+// 
+// Parameters:  None
+// 
+// *****************************************************
 void setup( void )
 {
-
     // duty Scalers good for 0x05
-    //SLADutyMax = 50;
-    //SLADutyMin = 5;
-    //SLRDutyMax = 50;
-    //SLRDutyMin = 20;
-
+    //SLA_duty_max = 50;
+    //SLA_duty_min = 5;
+    //SLR_duty_max = 50;
+    //SLR_duty_min = 20;
 
     // duty Scalers good for 0x02
-    SLADutyMax = 225;
-    SLADutyMin = 100;
-    SLRDutyMax = 225;
-    SLRDutyMin = 100;
+    SLA_duty_max = 225;
+    SLA_duty_min = 100;
+    SLR_duty_max = 225;
+    SLR_duty_min = 100;
 
     // set the PWM timers, above the acoustic range
-    TCCR3B = (TCCR3B & 0xF8) | 0x02; // pins 2,3,5 | timer 3
-    TCCR4B = (TCCR4B & 0xF8) | 0x02; // pins 6,7,8 | timer 4
-
+    TCCR3B = ( TCCR3B & 0xF8 ) | 0x02; // pins 2,3,5 | timer 3
```

Comment:
It's probably worth it to create a set_pwm function int16_t set_pwm( int16_t timer, int16_t bits ).  Not sure about this, it's setting the PWM mode with this step.